diff --git a/dist/Styles/Styles.svelte b/dist/Styles/Styles.svelte
index a611c9a54311971b6a3632cf5bf12196114ffbd5..51d2ce1a976106b8a7027d98d5a2f0a2d20afa7e 100644
--- a/dist/Styles/Styles.svelte
+++ b/dist/Styles/Styles.svelte
@@ -12,9 +12,9 @@
 </script>
 
 <svelte:head>
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" />
+  <!--<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" />-->
 
   {#if icons}
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css" />
+    <!--<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css" />-->
   {/if}
 </svelte:head>
diff --git a/src/Styles/Styles.svelte b/src/Styles/Styles.svelte
index a611c9a54311971b6a3632cf5bf12196114ffbd5..51d2ce1a976106b8a7027d98d5a2f0a2d20afa7e 100644
--- a/src/Styles/Styles.svelte
+++ b/src/Styles/Styles.svelte
@@ -12,9 +12,9 @@
 </script>
 
 <svelte:head>
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" />
+  <!--<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" />-->
 
   {#if icons}
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css" />
+    <!--<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.2/font/bootstrap-icons.min.css" />-->
   {/if}
 </svelte:head>
